### ISSUE TITLE
Translation question - resurgence not integrated

### ISSUE THREAD

**Issue by debyos** [osm-fr/osmose-backend#2025]
**Resurgence not integrated**

What is a **resurgence**? I can't find it anywhere on Osmose.
According to [Cambridge English Dictionary it means something](https://dictionary.cambridge.org/dictionary/english/resurgence) that is difficult to use here…

**Comment by frodrigo** [comment]
https://en.wikipedia.org/wiki/Spring_(hydrology)#Flow maybe not best definition ever.

For me is when water go underground ans then show up again. Maybe there is a better work for that in english.

It as actually tagged with like spring.
https://github.com/osm-fr/osmose-backend/blob/dev/analysers/analyser_merge_water_FR.py#L93

**Comment by debyos** [comment]
OK, now I understand.

It is a frequent error in Osmose that even the English part **is written in half-French**. In French [résurgence has your meaning](https://www.larousse.fr/dictionnaires/francais/r%C3%A9surgence/68794) and now it is pretty clear what it means. However, in English, resurgence is something completely different. (See: [false friends](https://en.wikipedia.org/wiki/False_friend) / [faux-ami](https://fr.wikipedia.org/wiki/Faux-ami))

At the beginning of the translation it was e.g. very difficult to find out what _integration_ means: not integrated, integration suggestion and so on. Now it seems that it may come from French as well. In English, you would say it in a completely other way, e.g. it is not mapped or it is missing from the map.

It is a huge work, I know, but perhaps it would be a good step to ask a native English speaker to revise it a bit…

**Comment by frodrigo** [comment]
> At the beginning of the translation it was e.g. very difficult to find out what integration means: not integrated, integration suggestion and so on. Now it seems that it may come from French as well. In English, you would say it in a completely other way, e.g. it is not mapped or it is missing from the map.

> It is a huge work, I know, but perhaps it would be a good step to ask a native English speaker to revise it a bit…

Yes, that a recurrent issue that translator fails on bad english. I make lot of this mistake myself, as a non native english speaker.

I always on favour of fixing english version.

**Comment by Famlam** [comment]
Unfortunately I have no clue how such a thing is translated into English: plenty of options including the current one: https://context.reverso.net/translation/french-english/r%C3%A9surgence+d%27eau
